DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py

```diff
@@ -0,0 +1,183 @@
+  )
+StandardSpecifications1D.append(
+      Specification(OverlayCurvesForTiming).groupBy("PXForward/OnlineBlock") # per-layer with history for online
+                     .groupBy("PXForward", "EXTEND_Y")
+                     .save()
+  )
+  
+StandardSpecifications1D_Num.append(
+      Specification(OverlayCurvesForTiming).groupBy("DetId/Event") # per-layer with history for online
+                               .reduce("COUNT")
+                               .groupBy("PXBarrel/PXLayer/OnlineBlock") 
+                               .groupBy("PXBarrel/PXLayer", "EXTEND_Y")
+                               .save()
+  )
+
+StandardSpecifications1D_Num.append(
+      Specification(OverlayCurvesForTiming).groupBy("DetId/Event") # per-layer with history for online
+                               .reduce("COUNT")
+                               .groupBy("PXForward/PXDisk/OnlineBlock") 
+                               .groupBy("PXForward/PXDisk", "EXTEND_Y")
+                               .save()
+  )
+
+#StandardSpecifications1D_Num.append(
+#      Specification(OverlayCurvesForTiming).groupBy("PXBarrel/OnlineBlock/PXLayer/Event") # per-layer with history for online
+#                     .reduce("COUNT")
+#                     .groupBy("PXBarrel/OnlineBlock") 
+#                     .groupBy("PXBarrel", "EXTEND_Y")
+#                     .save()
+#  )
+
+
+StandardSpecifications1D_Num.append(
+Specification(OverlayCurvesForTiming).groupBy("DetId/Event")
+                     .reduce("COUNT")
+                     .groupBy("PXBarrel/OnlineBlock")
+                     .groupBy("PXBarrel", "EXTEND_Y")
+                     .save()
+)
+
+
+StandardSpecifications1D_Num.append(
+Specification(OverlayCurvesForTiming).groupBy("DetId/Event")
+                     .reduce("COUNT")
+                     .groupBy("PXForward/OnlineBlock")
+                     .groupBy("PXForward", "EXTEND_Y")
+                     .save()
+)
+
+#StandardSpecifications1D_Num.append(
+#      Specification(OverlayCurvesForTiming).groupBy("PXForward/OnlineBlock/PXDisk/Event") # per-layer with history for online
+#                     .reduce("COUNT")
+#                     .groupBy("PXForward/OnlineBlock") 
+#                     .groupBy("PXForward", "EXTEND_Y")
+#                     .save()
+#  )
+#
+  
+# To Configure Phase1 DQM for Phase0 data
+SiPixelPhase1Geometry.upgradePhase = 0
+SiPixelPhase1Geometry.onlineblock = 5
+# Turn on 'online' harvesting. This has to be set before other configs are 
+# loaded (due to how the DefaultHisto PSet is later cloned), therefore it is
+# here and not in the harvestng config.
+DefaultHisto.perLumiHarvesting = True
+DefaultHistoDigiCluster.perLumiHarvesting = True
+DefaultHistoSummary.perLumiHarvesting = True
+DefaultHistoTrack.perLumiHarvesting = True
+
+
+# Pixel Digi Monitoring
+from DQM.SiPixelPhase1Digis.SiPixelPhase1Digis_cfi import *
+SiPixelPhase1DigisAnalyzer.src = cms.InputTag("siPixelDigis") # adapt for real data
```

Comment:
Hi @fioriNTU - this does not need to be here (as it does not change anything - if it did, then you would need to clone the module) - please remove it